Bio/PDB/parse_pdb_header.py

```diff
@@ -246,8 +250,23 @@ def _parse_pdb_header_list(header):
                 try:
                     dict['resolution'] = float(r)
                 except:
+<<<<<<< HEAD
```

Comment:
It looks like you committed a merge without resolving it - you'll need to clean up this in order for the Python code to work.